pkg/kapp/cmd/app/deploy.go

```diff
@@ -434,6 +434,10 @@ func (o *DeployOptions) calculateAndPresentChanges(existingResources,
 		changeViews := ctlcap.ClusterChangesAsChangeViews(clusterChanges)
 		changeSetView := ctlcap.NewChangeSetView(
 			changeViews, conf.DiffMaskRules(), o.DiffFlags.ChangeSetViewOpts)
+
+		if o.DiffFlags.ChangesYAML {
+			changeSetView.PrintCompleteYamlToBeApplied(o.ui, conf)
```

Comment:
I am thinking if we should have this flag for delete command as well, as there can be various strategies used while deleting which are highlighted by the `diff-changes-yaml` flag. (The `diff-changes` flag is also available for delete command)

Reply:
Updated